### PR TITLE
Start separating language agnostic NUG content

### PR DESCRIPTION
**Pull request by ethanrd**

### CODE REVIEW

**Comment by DennisHeimbigner** [comment]
Zarr does not add a String type (yet)

**Comment by DennisHeimbigner** [comment]
Other comments:
1. netcdf-c/ncgen/ncgen.1 has a CDL BNF if you need it

2. ````Is the "2 GiBytes size limit" in CDF-1 part of the data model?```` only if typesizes are defined for the atomic types
3.  ````Is the "only one unlimited dimension" limitation in CDF-1, 2, and -5 part of the data model?```` I would say yes
4. is a VLEN dimensions the same as unlimited?
5. Are coordinate variables actually part of the model or just a convention?

File: NUG-new/pages/nug_conventions.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: NUG Conventions
+last_updated: 2021-04-06
+sidebar: nnug_sidebar
+toc: false
+permalink: nug_convensions.html
```
Comment (WardF) [inline NUG-new/pages/nug_conventions.md:6]: Typo, should permalink be `nug_conventions.html` ?

Comment (ethanrd) [reply NUG-new/pages/nug_conventions.md]: Thanks @WardF for catching that. Just pushed fix.

**Pull request merged**